setup.py

```diff
@@ -34,6 +37,18 @@
                'Topic :: Software Development :: Quality Assurance',
                'Topic :: Software Development :: Testing']
 
+py_version = sys.version[:5]
```

Comment:
Let's revert these changes. I'll always be using Python 3.6+ to deploy changes.

Reply:
okay but some builds maybe fail